Tests/UnitTests/Android/app/src/main/cpp/CMakeLists.txt

```
@@ -18,7 +19,7 @@ add_library(UnitTestsJNI SHARED
     JNI.cpp
     ${UNIT_TESTS_DIR}/Shared/Shared.h
     ${UNIT_TESTS_DIR}/Shared/Shared.cpp)
-add_definitions(-DJSRUNTIMEHOST_PLATFORM="${JSRUNTIMEHOST_PLATFORM}")
+target_compile_definitions(UnitTestsJNI PUBLIC -DJSRUNTIMEHOST_PLATFORM="${JSRUNTIMEHOST_PLATFORM}")
```

Should this be public? Also, `-D` isn't necessary.

```suggestion
target_compile_definitions(UnitTestsJNI PRIVATE JSRUNTIMEHOST_PLATFORM="${JSRUNTIMEHOST_PLATFORM}")
```

---

Fixed.